scripts/combine-hybrid-results.py

```diff
@@ -0,0 +1,119 @@
+import sys
+from server.api.cvrs import cvr_contests_metadata
+from server.api.rounds import cvrs_for_contest, sampled_ballot_interpretations_to_cvrs, samples_not_found_by_round, contest_results_by_round, round_sizes
+from server.audit_math import sampler_contest, suite
+from server.audit_math.bravo import compute_cumulative_sample
+from server.models import *  # pylint: disable=wildcard-import
+from server.database import db_session
+
+
+def ballot_comparison_stratum(audit: Election):
+    contest = audit.contests[0]
+    vote_counts = {choice.id: choice.num_votes for choice in contest.choices}
+    suite_contest = sampler_contest.from_db_contest(contest)
+    reported_results = cvrs_for_contest(contest)
+    sample_results = sampled_ballot_interpretations_to_cvrs(contest)
+    sample_size = sum(ballot["times_sampled"] for ballot in sample_results.values())
+    misstatements = suite.misstatements(
+        suite_contest, reported_results, sample_results,
+    )
+    return suite.BallotComparisonStratum(
+        contest.total_ballots_cast, vote_counts, misstatements, sample_size,
+    )
+
+def ballot_polling_stratum(audit: Election, remap):
+    contest = audit.contests[0]
+    vote_counts = {choice.id: choice.num_votes for choice in contest.choices}
+
+    remapped_vote_counts = {remap[id1]: vote_counts[id1] for id1 in vote_counts}
+    suite_contest = sampler_contest.from_db_contest(contest)
+
+    sample_results = contest_results_by_round(contest)
+    remapped_sample_results = {}
+
+    for rd in sample_results:
+        remapped_sample_results[rd] = {}
+        for id1 in sample_results[rd]:
+            remapped_sample_results[rd][remap[id1]] = sample_results[rd][id1]
```

Comment:
If you want to get fancy with dictionary comprehensions...
```suggestion
    sample_results = { 
      round_id: {
        remap[choice_id]: result
        for choice_id, result in round_results.items()
      }
      for round_id, round_results in contest_results_by_round(contest).items()
    }
```

Reply:
ah I knew there was a way to do this I just couldn't figure it out.